src/base/projects/Stat.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  export let title: string | undefined = undefined;
+  export let id: string | undefined = undefined;
+  export let active = false;
+  export let clickable = false;
+  export let disabled = false;
+</script>
+
+<style>
+  .stat {
+    font-family: var(--font-family-monospace);
+    font-size: var(--font-size-tiny);
+    padding: 0.5rem 0.75rem;
+    padding-bottom: 1rem; /* moving the content a tad higher to match the previous span usage */
+    height: 2rem;
+    background: var(--color-foreground-1);
+    border: none;
+    color: var(--color-foreground);
+    border-radius: var(--border-radius-small);
+    min-width: max-content;
+  }
+  .active {
+    color: var(--color-background);
+    background: var(--color-foreground) !important;
+    background-color: var(--color-foreground);
+  }
+  .clickable {
+    cursor: pointer;
+  }
+  .clickable:hover {
+    background-color: var(--color-foreground-2);
+  }
+  .not-allowed {
+    cursor: not-allowed;
+    color: var(--color-foreground-5);
+  }
+</style>
+
+<button
+  {id}
```

Comment:
I'd [avoid using `id`s](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) as selectors for cypress.

I'd much rather we use [`aria-label`s](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) for that, this is also the preferred way in Playwright.

Reply:
AFAIK we don't want to use `data-*` labels, but I think `aria-label` should be ok. Will change that

Reply:
e7b75d8c43da5a4e8f908def84108938d9b1eb68